### PR TITLE
Clean database providers list with new mcloud aiida profile migration

### PR DESCRIPTION
**Pull request by unkcpz**

### CODE REVIEW

File: OPTIMADE-Client.ipynb [hunk 1]
```diff
@@ -44,10 +44,10 @@
     "    \"necro\",\n",
     "    \"jarvis\",\n",
     "]\n",
-    "skip_databases = {\"Materials Cloud\": [\"optimade-sample\", \"li-ion-conductors\", \"threedd\", \"sssp\"]}\n",
+    "skip_databases = {\"Materials Cloud\": [\"optimade-sample\"]}\n",
```
Comment (eimrek) [inline OPTIMADE-Client.ipynb:47]: currently, `li-ion-conductors` is not online (some mongodb issue), but eventually the plan is to put it online again. As i understood, it's meant as a testing database. Maybe best to keep it skipped.

File: README.md [hunk 2]
```diff
@@ -4,6 +4,10 @@ An OPTIMADE Client as a Materials Cloud tool.
 
 It runs a Jupyter notebook in Voilà with the [Materials Cloud Voilà template](https://github.com/materialscloud-org/voila-materialscloud-template).
 
+The test deployment is on https://optimadeclient.matcloud.xyz/, the production deployment is on https://www.materialscloud.org/work/tools/optimadeclient.
```
Comment (eimrek) [inline README.md:7]: Here I would use either the direct dokku url for both (preferred imo), or the materials cloud url, where the dokku page is embedded via iframes.

So:

https://optimadeclient.matcloud.xyz/ and https://optimadeclient.materialscloud.io/

or

https://dev-www.materialscloud.org/work/tools/optimadeclient and https://www.materialscloud.org/work/tools/optimadeclient

Comment (unkcpz) [reply README.md]: Agree! thanks.

File: OPTIMADE-Client.ipynb (hunk 1 above)
Comment (unkcpz) [reply OPTIMADE-Client.ipynb]: Changed.

**Review by eimrek — APPROVED** [review]

**Pull request merged**